sklearn/cross_decomposition/pls_.py

```diff
@@ -432,27 +432,24 @@ def predict(self, X, copy=True):
         Ypred = np.dot(X, self.coef_)
         return Ypred + self.y_mean_
 
-    def fit_transform(self, X, y=None, **fit_params):
```

Comment:
Why are you removing **fit_params?

It looks like it's not available in fit, so it's strange that it is here. Still, removing it *could* break someone's code